docs/CDCDatabase-streamingsource.md

```diff
@@ -10,21 +10,23 @@ CDC source produces messages in CDC format.
 Useful links:
 * [Goldengate site](https://www.oracle.com/middleware/technologies/goldengate.html)
 * [Installing Oracle GoldenGate](https://docs.oracle.com/goldengate/1212/gg-winux/GIORA/install.htm#GIORA162).
+* [Using Oracle GoldenGate for Oracle Database](https://www.oracle.com/pls/topic/lookup?ctx=en/middleware/goldengate/core/18.1&id=GGODB-GUID-110CD372-2F7E-4262-B8D2-DC0A80422806).
 
 Properties
 ----------
 **Reference Name**: Name used to uniquely identify this source for lineage, annotating metadata, etc.
 
-**Hostname**: Hostname of the SQL Server from which the data needs to be offloaded. 
-Ex: mysqlserver.net or 12.123.12.123.
+**Kafka Broker**: Kafka broker specified in host:port form. For example, example.com:9092.
 
-**Port**: SQL Server Port.
+**Kafka Topic**: Name of the topic to which Golden Gate publishes the DDL and DML changes.
 
-**Username**:  Username to use to connect to the specified database. Required for databases that need authentication. 
-Optional for databases that do not require authentication.
+**Default Initial Offset**: The default initial offset to read from. An offset of -2 means the smallest offset. 
+An offset of -1 means the latest offset. Defaults to -1. Offsets are inclusive. 
```

Comment:
what does smallest or latest offset mean?

Reply:
smallest means from the beginning. Latest means from recent available topic offset.

Reply:
can we add this in the text?

Reply:
but current terminology corresponds to Kafka documentation https://kafka.apache.org/documentation/ (auto.offset.reset property for example)

Reply:
It's still not clear for someone unfamiliar with the terminology

Reply:
fixed